mitiq/calibration/tests/test_calibration.py

```diff
@@ -38,9 +38,10 @@
             "circuit_depth": 1,
         },
         {
-            "circuit_type": "mirror",
-            "num_qubits": 2,
-            "circuit_depth": 1,
+            "circuit_type": "rotated_rb",
+            "num_qubits": 1,
```

Comment:
In the code, you are branching off logic based on the number of qubits. You want at least one test for each of the two logical branches.

Reply:
you are right!